CollectNetworkTrace/NetworkTraceConfiguration.cs

```diff
@@ -65,7 +65,7 @@ private static string GetEnvironmentVariable(string name)
         private static string GetConfigurationValue(string key)
         {
             var value = ConfigurationManager.AppSettings[key];
-            if (string.IsNullOrWhiteSpace(value))
+            if (string.IsNullOrWhiteSpace(value) || value.StartsWith("your", StringComparison.OrdinalIgnoreCase))
```

Comment:
There is a small problem with this change. If the value is the default one in config, then the variable value will get assigned to EMPTY if the APP SETTING does not exist so the check that checks for string `your` just a few lines below will never get called. So do something like this

```
var envVar = GetEnvironmentVariable(key);
if (!string.IsNullOrEmpty(envvar))
{
    value = envvar;
}
```

Reply:
Done. I fixed it but I used Environment.GetEnvironmentVariable(key) instead as I was getting the "Failed to read environment variable" exception. I tested it and it worked.